config/locales/user_mailer/es.yml

```
@@ -27,12 +27,11 @@ es:
         web gubernamentales participantes que utilizan %{app}, puede crear una
         nueva cuenta %{app} con la misma dirección de correo electrónico después
         de que se elimine su cuenta.<br><br>
-      subject: Elimina tu cuenta login.gov
+      subject: Elimina tu cuenta Login.gov
     account_reset_request:
-      cancel: '¿No quieres eliminar tu cuenta? Inicie sesión en su cuenta login.gov
-        para cancelar.'
-      header: Su cuenta será eliminada en 24 horas
-      intro_html: 'Como medida de seguridad, %{app} requiere un proceso de dos pasos
+       cancel: '¿No quieres eliminar tu cuenta? Inicie sesión en su cuenta Login.gov para cancelar.'
+       header: Su cuenta será eliminada en 24 horas
+       intro_html: 'Como medida de seguridad, %{app} requiere un proceso de dos pasos
```

Looks like this is indented by an extra space now.

```suggestion
      cancel: '¿No quieres eliminar tu cuenta? Inicie sesión en su cuenta Login.gov para cancelar.'
      header: Su cuenta será eliminada en 24 horas
      intro_html: 'Como medida de seguridad, %{app} requiere un proceso de dos pasos
```